CHANGELOG.md

```diff
@@ -5,9 +5,10 @@
 ### New Features
 
 - **AWS Mobile Client**
-  - **Breaking API Change** `SignUpResult` available in the user callback provided during sign up now contains UserSub(UID)
+  - **Breaking API Change**
+    - `SignUpResult` available in the user callback provided during sign up now contains UserSub(UID)
+    - Deprecated APIs `getCredentialsProvider`, `setCredentialsProvider`, `initialize(Context)` and `initialize(Context, AWSStartupHandler)` have been removed
   - AWSMobileClient now supports Cognito Custom Authentication flow. See [documentation](https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-authentication-flow.html#amazon-cognito-user-pools-custom-authentication-flow) for details
```

Comment:
Shouldn't we link to the Amplify API docs as well?

Reply:
Added link to the Amplify docs as well.